### ISSUE TITLE
Attachments produce no longer stored error message

### ISSUE THREAD

**Issue by jasonwhat** [notable/notable#555]
- **ChromeOS 73 Beta**:
- **Notable Version: 1.3.0**:

### Current behavior

I have my notes in Google Drive mounted through ChromeOS/Linux setup on a Pixel Slate.
1. Create a note
2. Select attachment 
3. Click attachment once it is on note
4. Get error: This attachment is no longer stored in disk

I've tried this with attachments that I first put it in the attachments directory and also with ones in other directories. I've had a few times where it works if it is in attachment directory already, but for the most part they simply do not and I get the error.

### Expected behavior

Attach file to note.
Click file and it opens.

**Comment by fabiospampinato** [comment]
Is the attachment actually getting copied to the attachments directory after you select it?

**Comment by jasonwhat** [comment]
> Is the attachment actually getting copied to the attachments directory after you select it?

No, it doesn't get copied. It shows that it is attached in the interface with the paperclip icon and file name if I click. However, I just tested again and if I put the attachment in the attachments directory  manually first it does seem to work most of the time. But it definitely will not copy a file into that directory. This could be an issue with how Google is doing files between Linux and ChromeOS.

**Comment by fabiospampinato** [comment]
> if I put the attachment in the attachments directory manually first it does seem to work most of the time

"Most of the time" doesn't sound good 🤔 

> This could be an issue with how Google is doing files between Linux and ChromeOS.

Maybe, I don't know anything about ChromeOS.

It would be great if somebody could help me debug this issue.

**Comment by jasonwhat** [comment]
Having tested a few times it seems that one issue is that the directory doesn't seem writeable. This may be an exclusive issue to ChromeOS and how they are handling Linux folder structure. 

Here is how I can successfully attach a file.
1. Move a file to attachments directory.
2. Wait for it to sync (directory is in Gdrive in this case)
3. Restart Notable
4. Attach file in Notable as you normally would.
